tailwind.config.mjs

```diff
@@ -1,14 +1,20 @@
 /** @type {import('tailwindcss').Config} */
 export default {
   content: ["./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}"],
+  darkMode: "class",
   theme: {
     extend: {
       colors: {
         primary: "hsl(var(--color-primary) / <alpha-value>)",
         secondary: "hsl(var(--color-secondary) / <alpha-value>)",
         hover: "hsl(var(--color-hover) / <alpha-value>)",
+        background: "hsl(var(--background))",
+        foreground: "hsl(var(--foreground))",
       },
     },
   },
+  variants: {
+    typography: ["dark"],
+  },
```

Comment:
Why is this needed?

Reply:
without this the prose invert won't work for dark mode , I searched and in a blog found this solution . Before using that I was not getting required result .

Reply:
Ahaa, cool!

Reply:
This worked fine for me without the typography options. What problem were you facing without this?

Reply:
In dark mode the change for prose classes were not applying as expected .